models/public/face-detection-retail-0044/face-detection-retail-0044.md

```diff
@@ -0,0 +1,56 @@
+# face-detection-retail-0044
+
+## Use Case and High-Level Description
+
+Face detector based on SqueezeNet light (half-channels) as a backbone with a
+single SSD for indoor/outdoor scenes shot by a front-facing camera. The backbone
+consists of fire modules to reduce the number of computations. The single SSD
+head from 1/16 scale feature map has nine clustered prior boxes.
+
+## Example
+
+![](./face-detection-retail-0001.png)
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| AP ([WIDER](http://shuoyang1213.me/WIDERFACE/))               | 83.00%                  |
+| GFlops                                                        | 1.067                   |
+| MParams                                                       | 0.588                   |
+| Source framework                                              | Caffe*                  |
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve. All numbers were evaluated by taking into account only faces bigger than
+60 x 60 pixels.
+
+## Performance
+
+## Inputs
```

Comment:
For public models descriptions of both original and converted model are required.
*Hereinafter.*